src/test/java/seedu/address/logic/parser/LogCommandParserTest.java

```diff
@@ -28,8 +28,30 @@ public void parse_validArgs_returnsLogCommand() {
     @Test
     public void parse_invalidFormat_failure() {
         Name name = STARBUCKS.getName();
+
+        // Starbucks hr/5
         String userInput = " " + name.fullName + " " + PREFIX_HOURS + "5";
         assertParseFailure(parser, userInput, String.format(MESSAGE_INVALID_COMMAND_FORMAT, LogCommand.MESSAGE_USAGE));
+
+        // -o n/Starbucks
+        String invalidOverrideHourMissing = " " + PREFIX_NAME + name.fullName;
+        assertParseFailure(parser, invalidOverrideHourMissing, String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                LogCommand.MESSAGE_USAGE));
+
+        // -o hr/5
+        String invalidOverrideMissingName = " " + PREFIX_HOURS + "5";
+        assertParseFailure(parser, invalidOverrideMissingName, String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                LogCommand.MESSAGE_USAGE));
+
+        String invalidNoNameAndNotResetAll = "-o";
+        assertParseFailure(parser, invalidNoNameAndNotResetAll, String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                LogCommand.MESSAGE_USAGE));
+
+        // test preamble n/Starbucks -o
+        String invalidNamePresentPreamblePresentNotResetAll = " test preamble " + PREFIX_NAME + name.fullName + " -o";
+        assertParseFailure(parser, invalidNamePresentPreamblePresentNotResetAll, String.format(
+                MESSAGE_INVALID_COMMAND_FORMAT, LogCommand.MESSAGE_USAGE
+        ));
```

Comment:
Not sure if you wanna specify in the comments what particular parts are missing from the command